appinventor/appengine/src/com/google/appinventor/client/explorer/commands/DeleteFileCommand.java

```diff
@@ -144,40 +143,38 @@ public void onClick(ClickEvent event) {
       });
    
       if (projectRootNode instanceof YoungAndroidFormNode) {
-          final String abc1 = ((YoungAndroidFormNode) projectRootNode).getFormName();
-        	HorizontalPanel labelPanel = new HorizontalPanel();
-          	Label warnmsg = new Label(MESSAGES.reallyDeleteWarning(abc1));
-  		labelPanel.add(warnmsg);
-  		labelPanel.setSize("100%", "22px");
-  		contentPanel.add(labelPanel);
-  		contentPanel.add(newNameTextBox);
-          	HorizontalPanel buttonPanel = new HorizontalPanel();
-  		HorizontalPanel checkboxPanel = new HorizontalPanel();
-  		buttonPanel.add(cancelButton);
-  		buttonPanel.add(okButton);
-  		checkboxPanel.add(cb);
-  		checkboxPanel.setSize("100%", "20px");
-  		contentPanel.add(checkboxPanel);
-  		buttonPanel.setSize("100%", "24px");
-  		contentPanel.add(buttonPanel);
-  		contentPanel.setSize("320px", "100%");
-  		okButton.setEnabled(false); 
-		newNameTextBox.getTextBox().addKeyUpHandler(new KeyUpHandler() {
-    		  @Override
-    		  public void onKeyUp(KeyUpEvent event) {
-      		    if(newNameTextBox.getText().equals(abc1)) {
-                	okButton.setEnabled(true);
-			newNameTextBox.setColor("#00c8ff");
-		    }	  
-        	    else
-        	    {
-                	okButton.setEnabled(false);
-			newNameTextBox.setColor("red");    
-        	    }
-    		  }
-  		});	     
-	  }
-
+        final String abc1 = ((YoungAndroidFormNode) projectRootNode).getFormName();
+        HorizontalPanel labelPanel = new HorizontalPanel();
+        Label warnmsg = new Label(MESSAGES.reallyDeleteWarning(abc1));
+  	labelPanel.add(warnmsg);
+  	labelPanel.setSize("100%", "22px");
+  	contentPanel.add(labelPanel);
+  	contentPanel.add(newNameTextBox);
+        HorizontalPanel buttonPanel = new HorizontalPanel();
```

Comment:
Hmm. The whitespace here is still weird. Are you using tab key for spacing? You're probably better off using spaces instead.

Reply:
Sorry actually i was using tab key instead of spaces, i will keep it in the mind to use spaces instead.
Removed all weird whitespaces